pkg/kubectl/describe.go

```diff
@@ -842,21 +842,25 @@ func describeContainers(label string, containers []api.Container, containerStatu
 		if len(resourceToQoS) > 0 {
 			fmt.Fprintf(out, "    QoS Tier:\n")
 		}
-		for resource, qos := range resourceToQoS {
+		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
```

Comment:
Yes. We should do that. I can post a patch for it later today, unless we
want to make that change in this PR itself.

On Wed, May 25, 2016 at 2:43 PM, Derek Carr <notifications@github.com>
wrote:

> In pkg/kubectl/describe.go
> <https://github.com/kubernetes/kubernetes/pull/26046#discussion_r64658693>
> :
>
> > @@ -842,21 +842,25 @@ func describeContainers(label string, containers []api.Container, containerStatu
> >  		if len(resourceToQoS) > 0 {
> >  			fmt.Fprintf(out, "    QoS Tier:\n")
> >  		}
> > -		for resource, qos := range resourceToQoS {
> > +		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
>
> I think we should stop reporting qos per resource in 1.3
>
> @bgrant0607 <https://github.com/bgrant0607> @vishh
> <https://github.com/vishh> - now that we have merged #14943
> <https://github.com/kubernetes/kubernetes/pull/14943> should we update
> the CLI to just output the QoS of the pod?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26046/files/c8a14e376e3df237d6d0d6a2d5266056cbfeab87#r64658693>
>